front-end/src/components/ServerIcon/ServerIcon.stories.tsx

```diff
@@ -0,0 +1,33 @@
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+
+import ServerIcon from "./ServerIcon";
+
+export default {
+  title: "Components/ServerIcon",
+  component: ServerIcon,
+} as ComponentMeta<typeof ServerIcon>;
+
+const Template: ComponentStory<typeof ServerIcon> = (args) => <ServerIcon {...args} />;
+
+export const Default = Template.bind({});
+Default.args = {};
```

Comment:
다른 스토리에서는 한 줄씩 다 띄웠는데, 어떻게 맞출지 논의해보시죠~

```suggestion
export const Default = Template.bind({});

Default.args = {};
```

Reply:
`*.stories.tsx`의 `args`인자에 한 줄 띄우는 것으로 통일하겠습니다 :)

Reply:
한 줄씩 다띄우는걸로 :+1: